### ISSUE TITLE
Specifying dash length list no longer works

### ISSUE THREAD

**Issue by APlowman** [plotly/plotly.py#1107]
In plotly.py 3.x, I can no longer specify a dashed line by specifying a string of dash lengths. Before updating to version 3, I could do this.

From the [docs ](https://plot.ly/python/reference/#scatter-line-dash) for `scatter -> line -> dash`:

> Sets the dash style of lines. Set to a dash type string ("solid", "dot", "dash", "longdash", "dashdot", or "longdashdot") or a dash length list in px (eg "5px,10px,2px,2px").

The latter option no longer appears to be valid, yielding a `ValueError`. For example, the following does not work (tested in a Jupyter notebook):

```python
import plotly.graph_objs as go

data = [
    {
        'y': [1,2,3],
        'line': {
            'dash': '5px,10px,2px,2px',
        }
    }
]
go.FigureWidget(data)
```

**Comment by jonmmease** [comment]
Hi @APlowman , thanks for the report!

I just opened an associated issue on the Plotly.js project to work out how we should handle this case. Check it out if you're interested in what went wronge here: https://github.com/plotly/plotly.js/issues/2903

**Comment by jonmmease** [comment]
Ok, looks like we'll handle this by introducing a custom `LineDash` validator class.

**Comment by jonmmease** [comment]
Fixed in #1136
